launch/tier4_system_launch/README.md

```diff
@@ -10,10 +10,14 @@ Please see `<exec_depend>` in `package.xml`.
 
 ## Usage
 
+Note that you should provide parameter paths as `PACKAGE_param_path`. The list of parameter paths you should provide is written at the top of `perception.launch.xml`.
+
 ```xml
   <include file="$(find-pkg-share tier4_system_launch)/launch/system.launch.xml">
     <arg name="run_mode" value="online"/>
     <arg name="sensor_model" value="SENSOR_MODEL"/>
+
+    <arg name="PACKAGE_param_path" value="..."/>
```

Comment:
Also, since you added `  <!-- Parameter files -->` and wrote parameters right under the `<launch>` tag, I think it's good to unify the formats of this example and the actual launch file.

Reply:
Resolved: https://github.com/autowarefoundation/autoware.universe/pull/2540/commits/57a3b7524b292c64f7fd4569506237994fdda583